### PR TITLE
Fix deprecation warnings for rspec

### PR DESCRIPTION
**Pull request by blackst0ne**

### CODE REVIEW

File: lib/capybara/webkit/matchers.rb [hunk 1]
```diff
@@ -7,11 +7,11 @@ module RspecMatchers
           actual.error_messages.any?
         end
 
-        failure_message_for_should do |actual|
+        failure_message do |actual|
```
Comment (mhoran) [inline lib/capybara/webkit/matchers.rb]: Perhaps we should alias these methods to retain compatibility with RSpec 2?

Comment (gabebw) [inline lib/capybara/webkit/matchers.rb]: Would doing `alias_method :failure_message :failure_message_for_should` be enough? Or should we check for its existence:

```ruby
if respond_to?(:failure_message)
  failure_message do
else
  failure_message_for_should do
end
```

Comment (mhoran) [inline lib/capybara/webkit/matchers.rb]: Hey @gabebw. I think `alias_method` would be just fine. That's the pattern I've seen elsewhere.

Comment (jferris) [inline lib/capybara/webkit/matchers.rb]: Will `alias_method` actually work? We're *calling* `failure_message_for_should`, not defining it.

Comment (mhoran) [inline lib/capybara/webkit/matchers.rb]: Great point. Upon further inspection of the example I saw previously, it's a bit more complicated than a simple `alias_method`:

```ruby
Spec::Matchers.constants.each do |c|
  if Class === (klass = Spec::Matchers.const_get(c))
    if klass.public_instance_methods.any? {|m| ['failure_message_for_should',:failure_message_for_should].include?(m)}
      klass.class_eval do
        alias_method :failure_message, :failure_message_for_should
      end
    end
    if klass.public_instance_methods.any? {|m| ['failure_message_for_should_not',:failure_message_for_should_not].include?(m)}
      klass.class_eval do
        alias_method :negative_failure_message, :failure_message_for_should_not
      end
    end
  end
end
```

**Comment by jferris** [comment]
@blackst0ne interested in making sure this is compatible with RSpec 2?

File: lib/capybara/webkit/matchers.rb (hunk 1 above)
Comment (runephilosof) [inline lib/capybara/webkit/matchers.rb]: How about the approach in PR #857. Does that work?

**Comment by jferris** [comment]
Thanks for submitting this, and sorry for the delay. I believe this was resolved by ed295f25bd05df4683820e44dd6bfb7c66ef151f.

**Pull request closed without merging**